test/e2e/conf.js

```diff
@@ -0,0 +1,17 @@
+exports.config = {
+  framework: 'jasmine',
+  seleniumAddress: 'http://localhost:4444/wd/hub',
+  jasmineNodeOpts: {
+    defaultTimeoutInterval: 250000,
+  },
+  specs: ['test.js'],
+
+  multiCapabilities: [
+    {
+      browserName: 'firefox',
```

Comment:
why don't we use headless browsers for testing? https://github.com/angular/protractor/blob/master/docs/browser-setup.md#using-headless-chrome